src/explore-education-statistics-common/src/components/Pagination.tsx

```diff
@@ -0,0 +1,122 @@
+import { ArrowLeft, ArrowRight } from '@common/components/ArrowIcons';
+import generatePageNumbers from '@common/components/util/generatePageNumbers';
+import { useDesktopMedia } from '@common/hooks/useMedia';
+import classNames from 'classnames';
+import React, { ReactNode } from 'react';
+
+interface Props {
+  currentPage: number;
+  nextPrevLinkRenderer: ({
+    children,
+    className,
+    pageNumber,
+    rel,
+  }: {
+    children: ReactNode;
+    className: string;
+    pageNumber: number;
+    rel: 'next' | 'prev';
+  }) => ReactNode;
+  pageLinkRenderer: ({
+    ariaCurrent,
+    ariaLabel,
+    className,
+    pageNumber,
+  }: {
+    ariaCurrent?: 'page' | undefined;
+    ariaLabel?: string;
+    className: string;
+    pageNumber: number;
+  }) => ReactNode;
+  totalPages: number;
+}
+
+const Pagination = ({
+  currentPage,
+  nextPrevLinkRenderer,
+  pageLinkRenderer,
+  totalPages,
+}: Props) => {
+  const { isMedia: isDesktopMedia } = useDesktopMedia();
+  const pageNumbers = generatePageNumbers({
+    currentPage,
+    totalPages,
+    offset: isDesktopMedia ? 2 : 1,
+  });
+
+  if (pageNumbers.length <= 1) {
+    return null;
+  }
+
+  return (
+    <nav
+      className="govuk-pagination"
+      role="navigation"
+      aria-label="Pagination navigation"
+    >
+      {currentPage !== 1 && (
+        <div className="govuk-pagination__prev">
+          {nextPrevLinkRenderer({
+            className: 'govuk-link govuk-pagination__link',
```

Comment:
Could refactor all instances of this `className` prop to use a single module scoped variable so we don't have to change it in multiple places in the future.